tests/common/layerrenderer.cpp

```diff
@@ -15,6 +15,12 @@
 */
 
 #include "layerrenderer.h"
+#if !USE_MINIGBM
+#include <drm.h>
```

Comment:
Will check. Should not be needed. Looks like some stale code form initial changes.